### ISSUE TITLE
There are some strings missing in the translation template

### ISSUE THREAD

**Issue by Ro-Borges** [oomek/attractplus#51]
There are quite a few strings missing from the file msg_template.txt

**Comment by oomek** [comment]
Yes, the language files and especially help messages need revising and updating. If you feel like you could spare some time and help I wolud really appreciate a pull request.

**Comment by Ro-Borges** [comment]
After I finish translating to  portuguese, I will try identify and locate the missing strings. From what I can tell, string that come from .nuts like plugins and intro can't be translated from the translation files, is that correct?
